src/TrimmedTodo.MinimalApi.Dapper.Sqlite/TrimmedTodo.MinimalApi.Dapper.Sqlite.csproj

```diff
@@ -10,6 +10,14 @@
     <UserSecretsId>7deba5fe-f06c-4a7d-9816-cdeccfc4e4a8</UserSecretsId>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="..\HelloWorld.Web\WebApplicationExtensions.cs" Link="WebApplicationExtensions.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="JwtConfigHelper.cs" />
```

Comment:
?

Reply:
Urgh VS was borking my includes for some reason, must've missed this one.